src/main/java/com/google/api/codegen/viewmodel/StaticLangApiMessageView.java

```diff
@@ -0,0 +1,95 @@
+/* Copyright 2017 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel;
+
+import com.google.api.codegen.SnippetSetRunner;
+import com.google.api.codegen.discovery.Schema;
+import com.google.auto.value.AutoValue;
+import java.util.List;
+import javax.annotation.Nullable;
+
+/**
+ * This ViewModel defines the structure of a Discovery doc's "schemas", "properties",
+ * "additionalProperties", and "items".
```

Comment:
Change the description here so that it is generically described as a message view model, and then mention Discovery as an example of how it can be used.